lib/TimeLimitedToken.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace OCA\OJSXC;
+
+class TimeLimitedToken {
+   public static function generateUser($node, $domain, $secret, $ttl = 60*60, $time) {
```

Comment:
I guess the $time parameter was added later, the optional parameter should be at the end.